src/main/java/org/jabref/gui/importer/actions/ResultHelper.java

```diff
@@ -0,0 +1,30 @@
+package org.jabref.gui.importer.actions;
+
+import org.jabref.gui.BasePanel;
+import org.jabref.gui.DialogService;
+import org.jabref.logic.importer.ParserResult;
+
+public class ResultHelper {
```

Comment:
make this an inner (private) class of opendatabaseaction, as this is an implementation detail

Reply:
I could get rid of this class and simplified the threading stuff